biogeophys/FatesHydroWTFMod.F90

```diff
@@ -341,7 +354,36 @@ function th_linear_res(this,psi) result(th)
   end function th_linear_res
 
   ! ===========================================================================
+  
+  subroutine get_min_ftc_weight(psi_min,psi,min_ftc_weight,dmin_ftc_weight_dpsi)
+    
+    ! This routine determines the weighting factor used
+    ! to generate a smooth curve to impose that min_ftc happens at min_psi
+    ! This method is to be used with any and all water transfer functions
+    
+    real(r8),intent(in) :: psi                  ! MPa suction
+    real(r8),intent(in) :: psi_min              ! Minimum value of psi we track, value that pins to ftc_min
+    real(r8),intent(out):: min_ftc_weight       ! weighting factor for min_ftc
+    real(r8),intent(out):: dmin_ftc_weight_dpsi ! derivative of weighting factor wrt psi
+
+    ! If the difference between psi and psi-min is greater than 10MPa
+    ! just assume there is no effect of the minimum function (ie weight 0)
+    min_ftc_weight = exp(min_ftc_scalar*max(psi_min-psi,-10._r8))
```

Comment:
When psi gets to about 2 MPA from psi_min, the weighting factor gets incredibly small, the 10 MPa here is just to avoid numerical problems with that exponent function.  We could test the min_ftc_scalar though.. I don't expect the choice of min_ftc_scalar to have any noticable impact on model output, but I could try a few different values to see if anything happens.  We have the utility parameter  [fates_dev_arbitrary](https://github.com/NGEET/fates/blob/main/parameter_files/fates_params_default.cdl#L720-L722) that could aid in this.